.github/workflows/pd-tests.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
   chunks:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
It may bring some resource loads of the GitHub action

Reply:
In the current situation in which the fail rate is high, fail-fast causes many sub-jobs canceled and rerun from the beginning after a `/retest`, then it has another chance to be failed or canceled. Before we fix the unstable tests, it only wastes more resources and significantly increases the failure rate of `/retest`.